### DISCUSSION TITLE
Disable MIDI echo/built in sounds

### DISCUSSION THREAD

**Discussion by nathan-stewart** [musescore/MuseScore#21199] [Q&A]
Is it possible to just completely disable Musescore4's laggy built in piano and use my MIDI hardware? Also is it possible to disable the MIDI loopback? That's perhaps the most annoying of them all. If I play a note on a MIDI keyboard, Musescore plays it back a few hundred miilseconds after hardware synth plays it. It shouldn't play it at all but there doesn't seem to be a way to disable this without losing the note playing while editing with the mouse.

**Comment by cbjeukendrup** [comment]
If there's a MIDI loop, that is caused by the combination of input and output device that you selected in Preferences > IO.

**Comment by nathan-stewart** [comment]
Okay - it's not technically loop - but  there doesn't seem to be a way to turn off playing MIDI notes received while still using MIDI for input. Musescore is SLOW and laggy even at 1024 samples buffer.  It's a noticeable slap back.

**Comment by nathan-stewart** [comment]
For that matter can I just completely disable Musescore's built in sounds and just use MIDI for everything?  Better sounds, better latency.  There doesn't seem to be a way to disable the sound output.  Wasn't this a thing in Musescore 3?

**Reply by cbjeukendrup** [reply]
You can disable playback during editing in Preferences > Note input. For the rest, you could mute channels in the Mixer (muting the master channel is currently not possible, but that will be fixed in the future; for now you can mute channels for individual instruments) (or just turn down the volume of your computer's speakers).
